cookbook/core/control_flow/chain_tasks.py

```diff
@@ -66,8 +60,39 @@ def chain_tasks_wf() -> pd.DataFrame:
 #
 #       create_node(task_name, parameter1=argument1, parameter2=argument2, ...)
 
+# %% 
+# How to chain SubWorkflows?
+# ^^^^^^^^^^^^^^^^^^^^^^^^
+# 
+# Similar to tasks, you can also chain `Subworkflows <https://docs.flyte.org/projects/cookbook/en/latest/auto/core/control_flow/subworkflows.html>`_ as follows:
+
+# %% 
+# First, define a sub workflow for ``write()``.
```

Comment:
Can we not explain this code cause it looks like repetition? Can we directly put the code without any content explaining it?

Reply:
Done.